src/diff.js

```diff
@@ -74,11 +74,84 @@ function minUnchanged(sizeA, sizeB) {
   return Math.min(15, Math.max(2, Math.floor(Math.max(sizeA, sizeB) / 10)))
 }
 
+const FENCED_NODE_NAMES = [
```

Comment:
Can you think of any node with inline content (heading, paragraph) that shouldn't be a fenced node? Or any nodes that have block or other level content that should be fenced?

Reply:
I can’t think of any node with inline content that shouldn’t be fenced, but we can double check with the current existing ones just to be sure